jena-fuseki2/jena-fuseki-ui/src/views/dataset/Edit.vue

```diff
@@ -218,9 +244,9 @@ export default {
     services (newVal) {
       if (newVal && newVal['gsp-rw'] && Object.keys(newVal['gsp-rw']).length > 0) {
         const element = this.$refs['graph-editor']
-        this.codemirrorEditor = CodeMirror.fromTextArea(element, this.cmOptions)
-        this.codemirrorEditor.on('change', cm => {
-          this.content = cm.getValue()
+        this.codemirrorEditor = editorFromTextArea(element, this.cmOptions.extensions)
+        EditorView.updateListener.of(v => {
```

Comment:
This was tough to understand how to fix. There was nothing in the docs of CodeMirror for migrating the code, no issues or anything obvious on their GH issue.

Luckily, found an example in CodeMirror forums for that: https://discuss.codemirror.net/t/listen-to-change-event/5095